diff --git a/vars.module b/vars.module
index 4c1f077..1cf3745 100644
--- a/vars.module
+++ b/vars.module
@@ -36,7 +36,7 @@ function vars_modules_uninstalled($modules) {
   foreach ($defaults as $default) {
     $name = $default->name;
 
-    if ($default->flags & self::VARS_DYNAMIC) {
+    if ($default->flags & Vars::VARS_DYNAMIC) {
       $name_like = db_like($name . '_') . '%';
       $vars = db_select('variable', 'v')
         ->condition(db_or()
